main.js

```diff
@@ -0,0 +1,30 @@
+$('.submit-post').on('click', function () {
+  var $input = $('#input-post').val();
+  var $name = $('#input-name').val();
+
+  $('.posts').append('<div> <button type="button" class="btn btn-link comment">comment</button> ' + $input + ' -Posted by: ' + $name + '<button type="button" class="btn btn-link remove">x</button> <div class="comment-section" style="display: none"> <div class="comments"> </div> <div ="comment-input-area"> <input type="text"class="form-control input-area input-comment" placeholder="Your comment"/> <input type="text"class="form-control input-area input-name2" placeholder="Your name"/> <button type="button" class="btn btn-primary submit-comment"> Submit Comment </button> </div>  </div> <hr /> </div> ');
+  bindRemove();
+});
+
+$('.posts').on('click', '.comment', function () {
+  var $commentSection =  $(this).closest('div').find('.comment-section');
+  if ($commentSection.css('display') ==='none') {
+    $commentSection.show();
+  }else {
+    $commentSection.hide();
+  }
+});
+
+$('.posts').on('click', '.submit-comment', function () {
+  var $commentInput = $(this).closest('.comment-section').find('.input-comment').val();
+  var $commentName = $(this).closest('.comment-section').find('.input-name2').val();
+  $(this).closest('.comment-section').find('.comments').append('<div> ' + $commentInput + ' -Posted by: ' + $commentName + '<button type="button" class="btn btn-link remove">x</button> </div>');
+  bindRemove();
+})
+
+
+var bindRemove = function () {
```

Comment:
I prefer that you use `on` for this - in any case maybe a better variable name would be `bindRemoveClick` because I was looking for this function to remove a binding, but really you're removing the click handler.